debian/usr.bin.snap-confine

```diff
@@ -69,6 +69,9 @@
 
     # reading seccomp filters
     /{tmp/snap.rootfs_*/,}var/lib/snapd/seccomp/profiles/* r,
+ 
+    # reading mount profiles
+    /{tmp/snap.rootfs_*/,}var/lib/snapd/mount/profiles/* r,
```

Comment:
The `profiles/` component was removed from the snapd side. Please update this here.